docs/geedocs/5.3.1/answer/7160008.html

```diff
@@ -40,18 +40,16 @@
 <h1><a href="../index.html">Google Earth Enterprise Documentation Home</a> | Release notes</h1>
 <h2>Release notes: Open GEE 5.3.1</h2>
 
-<p>Open GEE 5.3.1 is currently in development</p>
-
   <h5>
-    <p id="overview_5.3.1">New Features</p>
+    <h3 id="overview_5.3.1">New Features</h3>
```

Comment:
I don't think it's a good idea to have h3 inside an h5 block. You can probably just remove the h5 here

Reply:
done